migration-tests/pallets/staking/general.ts

```diff
@@ -0,0 +1,43 @@
+import '@polkadot/api-augment';
+import { MigrationTest, PostCheckContext, PreCheckContext, PreCheckResult } from '../../types.js';
+import assert from 'assert';
+import type { StorageKey } from '@polkadot/types/primitive';
+import type { Codec } from '@polkadot/types/types';
+
+export const generalStakingTests: MigrationTest = {
+    name: 'general_staking_pallet',
+    pre_check: async (context: PreCheckContext): Promise<PreCheckResult> => {
+        return {
+            rc_pre_payload: undefined,
+            ah_pre_payload: undefined
+        };
+    },
+    post_check: async (
+        context:    PostCheckContext,
+        pre_payload:    PreCheckResult
+    ): Promise<void> => {
+        const { rc_api_after, ah_api_after } = context;
+
+        const mode = await rc_api_after.query.stakingAhClient.mode();
+        assert(mode.toHuman() === 'Active', 'Assert staking mode is Active');
+
+        const activeEra = await ah_api_after.query.staking.activeEra();
+        assert(activeEra.isSome, 'Assert activeEra is Some');
+
+        const currentEra = await ah_api_after.query.staking.currentEra();
+        assert(activeEra.isSome, 'Assert activeEra is Some');
+
+        const currentMultiBlockPhase = await ah_api_after.query.multiBlockElection.currentPhase();
+        console.log('currentMultiBlockPhase is ', currentMultiBlockPhase.toHuman());
+
+        // TODO: check for session report. Is it an event or a query?
+        if (currentMultiBlockPhase.toHuman() === 'Off') {
```

Comment:
```suggestion
        if (currentMultiBlockPhase.toHuman() === 'Off') {
```

yeah this is the beauty of PJS not having the types for new pallets I guess :))